### DISCUSSION TITLE
Stutters

### DISCUSSION THREAD

**Discussion by mikecolive** [kubilus1/autoortho#59] [General]
Anyone else having stutters when using Autoortho? Sim isn't smooth anymore.

**Comment by tgcbraun** [comment]
I do have long hangs of over 2 minutes, generally associated with "WARNING:getortho:Failed getting: Chunk" errors

**Reply by kubilus1** [reply]
Probably caused by throttling from bing maps.  I have some stuff in the works to alleviate this to some degree.

**Reply by TT11MM** [reply]
Is it possible the stutters are happening when X-plane loads the next part of the scenery. I only get stutters when in cruise. With stutters I mean framedrops to approximately 10 fps for about 20-30 seconds. Just enough to kick me from Vatsim. 

Also, would you recommend any other map provider besides Bing? It feels to me Bing is by far the fastest server (initial loading).

**Comment by Trillionengineer** [comment]
I too have noticed performance has suffered. Before, I’d get drops from 50fps cruise to 30 fps min. Now it goes down to 12-20 fps. It stays that way for a couple minutes. I too get kicked off vatsim multiple times during cruise. Any suggestions? Should we try a different map provider? Running it on Xplane 11 windows 11. R9 5900Hs, RTX3060. (Asus g14 2021). Running it using Dokan. Before the performance decrease was with WinFSP. 

I have noticed load times have significantly improve though!

**Comment by lknfly1** [comment]
Yeap same here, now I'm flying my first journey IFR LFBZ/GMAD with Autoortho on IVAO, A319 Toliss, a 2H47 trip, but a lot of stutters and a few very short freezes unfortunately, every 5 min over Spain at FL360, IVAO alarms under 20 fps then getting back to 50 fps (cruising)... It is unbearable somehow. Hopefully it's going to get better over Morrocco!
Yesterday it was ok in VFR, slow speed, low altitude except when I reach the Alpes/Mountains...

Thank you for the dev to share autoortho and good to know that he's aware about the issue.
I'll wait for the next update, keep up the great work!

Cheers ;-)

**Reply by TT11MM** [reply]
For anybody who is reading this, for me it really helped to let Autoortho, ActiveSky, X-pilot (and other support software for X-plane) run on different physical cores than XPlane itself. I have this set up with  Process Lasso. 

I have X-plane running on cores 3-7, and programs mentioned above on core 8. X-plane barely uses multicore but requires 4 cores, mainly during initial loading.  

Framedrops got a lot better for me, usually the drops don’t get lower than 20-25 if they happen at all.

**Comment by lknfly1** [comment]
For information I did a couple of VFR flights the last three days from North Morocco to South, GA and A319 Toliss, now flying the 742 FELIS to Senegal, online right now on IVAO/Active Sky and I didn't encounter any stutter or freeze screen, only in Europe so far (didn't test USA/ASIA) 

The autoortho works very well in this big area, sure that it less dense than in Europe!

@TT11MM I think I will reinstall Process Lasso then! ;-)

![XP Bled](https://user-images.githubusercontent.com/132143466/236878285-82d9a01f-643f-459b-ba33-68e5787d64da.jpg)
![XP12 Maroc](https://user-images.githubusercontent.com/132143466/236878292-4b366524-6770-402b-952c-15894c782db1.jpg)
![PA-31 REP Morocco](https://user-images.githubusercontent.com/132143466/236878294-919aedb1-b2b7-412b-879a-0ff3d8f5a6df.jpg)

**Comment by loganlaporte** [comment]
I too can confirim using projectlasso and seperating autoortho to its own core/thread section does alleviate the vatsim disconnects. I still get the warning for framerate but it catches itself much faster now. More like a minor stutter now.
I ended up giving autoortho 3c/6t,

**Comment by zSan1** [comment]
My stuttering decreased (and recovery from low fps to normal was much faster) when selecting 2 physical cores / 2 logical cores for autoortho and 4 physical cores / 4 logical cores for xplane 12 in Process Lasso, I use a ryzen 5 3600. Looking at the process lasso, I saw that autoortho in a few moments uses 100% of the cores and then quickly the xplane also uses 100% of the cores causing stutter.

**Comment by gusram5** [comment]
Well, I have a Ryzen 9 5900X, and using Process Lasso I have assigned threads 0-12 to XP12 and 13-23 to Autoortho, I feel better performance, the stutters are reduced a lot and when they occur they happen very fast and the recovery is fast as well.
In general I feel I gained more FPS in XP12.

**Comment by nasiir24** [comment]
Hi everyone, I know I am late to this thread, but I recently downloaded AutoOrtho and have been having nothing but problems when cruising or landing. I was wondering if someone has figured out a solution to avoiding the freezing, stuttering, and disconnect from VATSIM. I've attached a copy of my recent log. Any help would greatly be appreciated. 
[Log 10-16-23.txt](https://github.com/kubilus1/autoortho/files/12923121/Log.10-16-23.txt)

**Reply by Holgi79** [reply]
I  am also affekcted by the FPS drops. I am using a 5800X 3D. Generelly get nice FPS. I tried Process Lasso but it forces rhe XP12 too crash :-(
